source/contribute.rst

```diff
@@ -41,128 +41,144 @@ contributing back to a Doctrine project.
 Initial Setup
 -------------
 
--  Setup a `GitHub <https://github.com>`_ account.
--  Fork the repository of the project you want to contribute to. In this example
-   it will be `DBAL <https://github.com/doctrine/dbal>`_
--  Clone your fork locally
+We are assuming you have cloned the project, have a Github account, and
+have forked the project. If any of this sounds unfamiliar, please `learn
+about that
+<https://docs.github.com/en/get-started/quickstart/contributing-to-projects>`_
+first.
+
```

Comment:
> Or to rename it… should be less confusing.

I think I misunderstood this, I thought you wanted to rename `origin` to something else.

I agree, there are people who clone the original repo and add their fork as a second remote, or they clone their fork and add `upstream` or similar.

Either way, wouldn't it be safe if we suggest to add a `doctrine` remote pointing to the original Doctrine project's repo, and use that remote name in all other git commands?

Reply:
I just see that in fact, you do that – rename `origin` to `doctrine`. That may be wrong, depending on how people set up their clone. And if they follow the GitHub instruction linked above, it will definetly be wrong, since those instructions end up with `origin`= the fork.

Reply:
@greg0ire I think this point still needs to be addressed:

Simply renaming `origin` to `doctrine` may not be correct. Depending on how people obtained their clone, `origin` may be their fork or the original (upstream) repo.

When closely following the instructions linked in the previous sentence, then `origin` will be the _fork_, so the rename suggested here is definetly wrong.